blocks/melody/ff_container/ff_container-landing-section/ff_container-landing-section.less

```diff
@@ -26,7 +26,8 @@
     &__more-link {
         #ff_mix-util-prose > .link-no-decoration();
```

Comment:
You can pass colour vars in to the link-no-decoration mixin; you shouldn't need to add the states here